components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -21,7 +21,9 @@ local getIconName = require('Module:IconName').luaGet
 
 local MatchGroupInput = Lua.import('Module:MatchGroup/Input', {requireDevIfEnabled = true})
 
-local ALLOWED_STATUSES = { 'W', 'FF', 'DQ', 'L' }
+local _STATUS_SCORE = 'S'
```

Comment:
Does this mean it has a score?

```suggestion
local _STATUS_HAS_SCORE = 'S'
```

Reply:
it means a numerical score was entered
if a score of the kind "W", "FF", ... was entered status is not set to "S"

Reply:
Alright, then I think my suggestion is clearer :)

Reply:
done